toy_2d/src/ilqr.py

```diff
@@ -258,12 +258,13 @@ def animate(self, init_state, controls, show_vis=False):
         states = self.lcs.system.state_history
         controls = self.lcs.system.control_history
         control_forces, control_locs = controls[:, :2], controls[:, 2:]
+        gif_name = "gif_" + str(self.test_num)
         # Generate a gif of the simulated rollout.
-        vis_utils.animation_gif_polytope(self.lcs.system.params.polytope, states, 'square', self.lcs.system.params.dt,
+        vis_utils.animation_gif_polytope(self.lcs.system.params.polytope, states, gif_name, self.lcs.system.params.dt,
             controls=(control_forces, control_locs), save=True)     
 
-    def save_contols(self, controls, iter):
-        file_name = "controls"
+    def save_contols(self, controls):
+        file_name = "controls_" + str(self.test_num)
```

Comment:
I like that the saved controls will necessarily have the same number in the file as the generated gif.  Likewise, I think it would be useful for the controls to have `ilqr` in the filename too -- e.g. `"ilqr_controls_" + str(self.test_num)`.